website/src/pages/main.tsx

```diff
@@ -85,66 +85,80 @@ export const Main: React.FC = () => {
+          discoverability through auto-completion and type errors, as well as
+          facilities for abstraction and easy sharing of code. Behind the
+          scenes, we translate your Typescript configuration into <a
+          href="https://nixos.org/">Nix</a>.
 
+        </div>
+
+        <div className="garn-description"> <b>garn</b> currently supports <b>go</b>, <b>javascript</b>, and <b>haskell</b>.
+        </div>
+      </section>
+      <section className="features">
+          <h1>features</h1>
 
           <div className="feature">
-            <h2>Declarative environments</h2>
-            <ul>
-              <li>Your dependencies go in config file, not in a README </li>
-              <li>Makes life easy for your collaborators</li>
-              <li>Unlike containers, combines well with your personal dev environment</li>
-              <li>Works on Linux and Macs</li>
-            </ul>
+            <h2>declarative environments</h2>
+            <div className="feature-description">
+              Specify your dependencies easily and precisely.
+              Then switch between environments with a single command. Unlike
+              containers, <b>garn</b> environments combine well with your personal
+              dev environment.
+            </div>
             <Asciinema src={envDemoCastUrl} />
           </div>
 
           <div className="feature">
-            <h2>Reproducible builds and tests</h2>
-            <ul>
-              <li>Builds and checks can be run with no further configuration on CI</li>
-              <li>No more Heisenbugs</li>
-              <li>Because builds are deterministic, they are cacheable, speeding up development and CI</li>
-            </ul>
+            <h2>reproducible builds and tests</h2>
+            <div className="feature-description">
+            Builds and checks are run in a sandboxed environment, ensuring that
+            they are reproducible and deterministic. They can then be run
+            without further work on CI, exactly as they are locally. And
+            results can be cached and shared among your CI and local environment.
+
+            </div>
             <Asciinema src={buildDemoCastUrl} />
           </div>
 
 
           <div className="feature">
-            <h2>A unified interface</h2>
-            <ul>
-              <li>A CLI that makes the action of your project obvious</li>
-              <li>The same commands can run any test in any repo</li>
-              <li>Discoverability exactly where you need it</li>
-            </ul>
+            <h2>a unified interface</h2>
+            <div className="feature-description">
+            <b>garn</b> provides a unified interface for all your project's
+            tests, builds, environments, and executables, as well as help
+            messages that describe exactly what commands are available.
+            Instead of wrangling with a host of tools, you can focus on getting
+            things done.
```

Comment:
This was hard to parse for me the first time I read it

```suggestion
            <b>garn</b> provides a unified interface for all your project's
            tests, builds, environments, and executables. It can also serve
            as command documentation, printing help
            messages that describe exactly what commands are available.
            Instead of wrangling with a host of tools, you can focus on getting
            things done.
```